audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAuditWriter.java

```diff
@@ -281,12 +287,15 @@ public void scheduledFlush() {
     }
 
     /**
-     * Scheduled cleanup of audit data in database.
+     * Scheduled cleanup of audit data in the database.
      */
-    @Scheduled(fixedDelayString = "${audit.cleanup.delay.fixed:3600000}", initialDelayString = "${powerauth.audit.cleanup.delay.initial:1000}")
+    @Scheduled(cron = "${audit.cleanup.cron:0 0 0/1 * * *}")
+    @SchedulerLock(name = "audit.cleanup", lockAtLeastFor = "1s", lockAtMostFor = "10m")
```

Comment:
Can we externalize the `lockAtLeastFor` and `lockAtMostFor` properties? And change the defaults should both be probably longer.

Reply:
A good idea!